js/main.js

```diff
@@ -1,48 +1,33 @@
 
 var $addCard = document.querySelector('#add-card');
-var $searchBarDiv = document.querySelector('#search-bar');
-var $cardCount = document.querySelector('#card-deck-count');
-var $infoText = document.querySelector('#click-to-add');
 var $noDeck = document.querySelector('#no-deck');
-var $newDeck = document.querySelector('.new-deck');
 var $search = document.querySelector('#search');
 var $myDeck = document.querySelector('#my-deck');
-var $myDeckView = document.querySelector('#my-deck-view');
 var $logo = document.querySelector('.logo');
 var $searchBar = document.querySelector('.search-bar');
-
-function searchBarAppears(event) {
-  event.preventDefault();
-  $newDeck.className = 'new-deck hidden';
-  $cardCount.className = 'minor-padding column-full row space-between hidden';
-  $searchBarDiv.className = 'colun-full row justify-center search-bar-padding';
-  $infoText.className = 'row justify-center align-center hidden';
-  $myDeckView.className = 'row justify-center align-center';
-}
-
-function myDeckAppears(event) {
-  event.preventDefault();
-  $newDeck.className = 'new-deck';
-  $cardCount.className = 'minor-padding column-full row space-between hidden';
-  $searchBarDiv.className = 'colun-full row justify-center hidden';
-  $infoText.className = 'row justify-center align-center hidden';
-  $myDeckView.className = 'row justify-center align-center';
+var $tabView = document.querySelectorAll('.tab-view');
+
+function switchView(dataView) {
+  for (var i = 0; i < $tabView.length; i++) {
+    if ($tabView[i].getAttribute('data-view') === dataView) {
+      $tabView[i].className = 'tab-view';
+    } else {
+      $tabView[i].className = 'tab-view hidden';
+    }
+  }
 }
 
-function newDeck(event) {
+function switchViewing(event) {
   event.preventDefault();
-  $newDeck.className = 'new-deck hidden';
-  $cardCount.className = 'minor-padding column-full row space-between';
-  $searchBarDiv.className = 'colun-full row justify-center hidden';
-  $infoText.className = 'row justify-center align-center';
-  $myDeckView.className = 'row justify-center align-center';
+  var dataView = event.target.getAttribute('data-view');
+  switchView(dataView);
 }
 
-$myDeck.addEventListener('click', myDeckAppears);
-$search.addEventListener('click', searchBarAppears);
-$noDeck.addEventListener('click', newDeck);
-$addCard.addEventListener('click', searchBarAppears);
-$logo.addEventListener('click', myDeckAppears);
+$myDeck.addEventListener('click', switchViewing);
+$search.addEventListener('click', switchViewing);
+$noDeck.addEventListener('click', switchViewing);
+$addCard.addEventListener('click', switchViewing);
+$logo.addEventListener('click', switchViewing);
```

Comment:
You should add a class to all of these elements so you can `querySelectorAll` for all of them, then loop over the list and add this event listener to every item with that class. That way if you want to add another element to your list of elements capable of view swapping, you just need to add that class. I recommend a class like `view-container`